docs/index.html

```diff
@@ -89,16 +70,14 @@
                 }
             }
         </style>
-        <div id="pre-bootstrap">
-            <div class="messaging">
-                <h1>
-                    <img src="./assets/TriFlame.svg" style="width:24px">
-                </h1>
-                <div class="logo-background-brand">
+                <div id="pre-bootstrap">
+                    <div class="messaging">
+                        <h1>
+                            <img src="./assets/TriFlame.svg" width="17" height="50"> <img src="./assets/CashmereAppLogo.svg" width="200" height="50">
```

Comment:
Thanks for doing this @AdamGMiller!  Just a couple adjustments to this line, the tri-flame should be a little bigger, and on a separate line from the logo (or else Paul will get mad that we're messing with the master brand):
`<img src="./assets/TriFlame.svg" width="24" height="41"><br><img src="./assets/CashmereAppLogo.svg" width="200" height="50">`